### DISCUSSION TITLE
小程序问题反馈

### DISCUSSION THREAD

**Discussion by mba1398** [datawhalechina/DOPMC#74] [建议反馈]
既然什么都可以谈，不妨把问题抛出来。

问题1：2022-07-30 发现小程序存在一个bug，水卡的同学依然可以收到结业证书，这显然不太合理，有开发同学说会尽快修复，期待中。

**Comment by mba1398** [comment]
问题2：小程序 owner 是谁？

**Reply by SuperSupeng** [reply]
好问题，近期要把各个项目小组的现阶段负责人梳理出来，发布社区的新版组织架构

**Reply by mba1398** [reply]
光城
